### ISSUE TITLE
Fix illegal reflective access

### ISSUE THREAD

**Issue by x3ntrix** [JAM-Software/SonarDelphi#23]
When running the plugin with Java > 8, it reports this warning:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.thoughtworks.xstream.core.util.Fields (file:/C:/Users/CIUser/.sonar/cache/a4ec2f3af08969606e86a41958fd4a7a/sonar-delphi-plugin.jar_unzip/META-INF/lib/xstream-1.4.11.jar) to field java.util.TreeMap.comparator
WARNING: Please consider reporting this to the maintainers of com.thoughtworks.xstream.core.util.Fields
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```
By default, only the first access is printed. Therefore, there could be more problems.
With Java >= 16 the access is denied by default and the code probably fails.

**Comment by x3ntrix** [comment]
Issue in the xstream repository: x-stream/xstream#101
